src/static/commandline.html

```diff
@@ -6,8 +6,7 @@
 	<link rel="stylesheet" href="css/commandline.css">
     </head>
     <body>
-        <div id="completions"></div>
-        <div id="command-line-holder"><span id="tridactyl-colon"></span><input id="tridactyl-input"></input></div>
-        <script src="../commandline_frame.js"></script>
+	<div id="tridactyl-ui-container"></div>
+	<script src="../commandline_frame_react.js"></script>
```

Comment:
this script is actually cliframe/index.tsx - specified in webpack.config.js - I'm not sure this subterfuge is a good idea...